documentation/docs/welcome.md

```diff
@@ -24,13 +24,9 @@ Use Stronghold to store account data of the `wallet.rs`. It integrates the best
 With the `wallet.rs` library, developers do not need to use a self-generated seed anymore. By default, `Stronghold` will create and store the encrypted seed at rest. It is not possible to extract the seed from `Stronghold` for security purposes. `Stronghold` uses encrypted snapshots that can easily be backed up and securely shared between devices. These snapshots are further secured with a password.
```

Comment:
We now don't auto generate the mnemonic anymore, because users should still do a backup of the mnemonic
```suggestion
`Stronghold` can store the encrypted seed at rest. It is not possible to extract the seed from `Stronghold` for security purposes. `Stronghold` uses encrypted snapshots that can easily be backed up and securely shared between devices. These snapshots are further secured with a password.
```